src/GitHub.App/ViewModels/RepositoryCloneViewModel.cs

```diff
@@ -80,7 +83,17 @@ public class RepositoryCloneViewModel : BaseViewModel, IRepositoryCloneViewModel
                 signalReset: filterResetSignal
             );
 
-            BaseRepositoryPathValidator = this.CreateBaseRepositoryPathValidator();
+            var baseRepositoryPath = this.WhenAny(
+                x => x.BaseRepositoryPath,
+                x => x.SelectedRepository,
+                (x, y) => x.Value);
+
+            BaseRepositoryPathValidator = ReactivePropertyValidator.ForObservable(baseRepositoryPath)
+                .IfNullOrEmpty("Please enter a repository path")
+                .IfTrue(x => x.Length > 200, "Path too long")
+                .IfContainsInvalidPathChars("Path contains invalid characters")
+                .IfPathNotRooted("Please enter a valid path")
+                .IfTrue(IsAlreadyRepoAtPath, Resources.RepositoryNameValidatorAlreadyExists);
```

Comment:
Instead of copying this here, maybe it's best to pass an observable to `CreateBaseRepositoryPathValidator`. The repository creation viewmodel also does these exact same checks, it would be nice not to duplicate code.